src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -91,13 +97,23 @@ private void init() {
     @Scheduled(fixedDelay = 30000)
```

Comment:
I know this is not a change in this ticket but: 
1. shouldn't this fixedDelay be parameterized?
2. is it every (reasonable) possible that kafkaProperties is null? do we gain anything by handling this silently?